### PR TITLE
Citation file was added

### PR DESCRIPTION
**Pull request by tubayildizli**
Citation file

### CODE REVIEW

File: CITATION.cff [hunk 1]
```diff
@@ -16,6 +16,6 @@ authors:
     given-names:Hanifa
 title: "Mandarin Project"
 version: 2.0.4
-doi:
+doi:xxxx
```
Comment (HolyBorito) [inline CITATION.cff:19]: Very nice!

**Review by HolyBorito — APPROVED** [review]
Very good!

**Review by HolyBorito — APPROVED** [review]
Good to go!

**Comment by tubayildizli** [comment]
@HolyBorito can you review it again?

**Pull request merged**